examples/models/phi-3-mini/main.cpp

```diff
@@ -6,9 +6,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+#include <executorch/extension/llm/runner/text_llm_runner.h>
```

Comment:
nit: may like to keep headers sorted: std lib first, then 1st party

Reply:
This is properly formatted (not sure why this way though)